### PR TITLE
Fix/Implementation #643

### PR DESCRIPTION
**Pull request by RoiEXLab**
This Fixes/Implements #643

---
This change is [Reviewable](https://reviewable.io/reviews/triplea-game/triplea/689)

### CODE REVIEW

**Comment by DanVanAtta** [comment]
Does the download maps window remain non-modal after this change? Asked another way, if the download window is open, can you still do stuff like chat on the lobby, etc?

File: src/games/strategy/engine/framework/map/download/DownloadMapsWindow.java [hunk 1]
```diff
@@ -72,23 +71,22 @@ public static void showDownloadMapsWindow() {
...
+    dia.setLocationRelativeTo(parent);
     dia.setMinimumSize(new Dimension(200, 200));
     dia.setVisible(true);
   }
-
-  private DownloadMapsWindow(final Optional<String> mapName, final List<DownloadFileDescription> games) {
-    super("Download Maps");
+  
+  private DownloadMapsWindow(final Optional<String> mapName, final List<DownloadFileDescription> games, JFrame parent) {
+    super(parent, "Download Maps", true);
```
Comment (RoiEXLab) [inline src/games/strategy/engine/framework/map/download/DownloadMapsWindow.java]: if this true is changed to false or removed the download maps windows will keep independent....

**Comment by RoiEXLab** [comment]
Currently no - but thats easy to achieve by simply removing this true boolean...

**Comment by DanVanAtta** [comment]
It's intentional for the download maps window to be non-modal. Meaning, you should not be forced to stare at progress bars, but instead can continue interacting with the game.

**Pull request merged**